mesos_master/datadog_checks/mesos_master/mesos_master.py

```diff
@@ -209,11 +210,8 @@ def check(self, instance):
         instance_tags = instance.get('tags', [])
         if instance_tags is None:
             instance_tags = []
-        default_timeout = self.init_config.get('default_timeout', 5)
```

Comment:
Removing `default_timeout = self.init_config.get('default_timeout', 5)` seems to be a breaking change if users rely on `default_timeout`.

Reply:
Yeah, I'd keep this line with a comment saying that is legacy but still remove it from the documentation. I dont think it makes sense to be able to override default timeout when you have timeout for that.

Reply:
Should be resolved with this PR https://github.com/DataDog/integrations-core/pull/4226